tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -107,11 +109,12 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1Cluster() {
 		name      string
 		nodeRoles []nodepools.NodeRoles
 		client    *rancher.Client
+		cluster   *management.Cluster
 	}{
-		{"1 Node all roles Admin User", nodeRoles0, r.client},
-		{"1 Node all roles Standard User", nodeRoles0, r.standardUserClient},
-		{"3 nodes - 1 role per node Admin User", nodeRoles1, r.client},
-		{"3 nodes - 1 role per node Standard User", nodeRoles1, r.standardUserClient},
+		{"1 Node all roles Admin User", nodeRoles0, r.client, nil},
```

Comment:
We're just adding nils, making this confusing. Let's remove cluster as a test parameter attribute.